stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/MessageApi.kt

```diff
@@ -0,0 +1,52 @@
+package io.getstream.chat.android.client.api2
+
+import io.getstream.chat.android.client.api.AuthenticatedApi
+import io.getstream.chat.android.client.api2.model.requests.MessageRequest
+import io.getstream.chat.android.client.api2.model.response.MessageResponse
+import io.getstream.chat.android.client.call.RetrofitCall
+import retrofit2.http.Body
+import retrofit2.http.DELETE
+import retrofit2.http.GET
+import retrofit2.http.POST
+import retrofit2.http.Path
+import retrofit2.http.Query
+
+@MoshiApi
+@AuthenticatedApi
+internal interface MessageApi {
```

Comment:
Example of a new Api interface, not sure how many of these we'll want (but more than the single one in the old impl)